Tests/UnitTests/Mocks/MockSubscriberAttributesManager.swift

```diff
@@ -178,6 +178,18 @@ class MockSubscriberAttributesManager: SubscriberAttributesManager {
         invokedSetMixpanelDistinctIDParametersList.append((mixpanelDistinctID, appUserID))
     }
 
+    var invokedSetFirebaseAppInstanceID = false
+    var invokedSetFirebaseAppInstanceIDCount = 0
+    var invokedSetFirebaseAppInstanceIDParameters: (firebaseAppInstanceID: String?, appUserID: String?)?
```

Comment:
I know you were following the pattern in other methods, but these aren't used?

Reply:
Thanks! We were missing a few tests that referenced these for the last several integrations- fixed

Reply:
`invokedSetFirebaseAppInstanceID` still isn't used. I feel like those are redundant with the "count" ones.

Reply:
Yeah, that makes sense. I think since all of the others follow the same pattern, I'll log this in jira to tackle them all at once 👍

Reply:
Yeah that makes sense, we could refactor this a lot to remove all the duplication.